src/resources/views/crud/columns/upload.blade.php

```diff
@@ -0,0 +1,8 @@
+@php
+    $column['escaped'] = $column['escaped'] ?? false;
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['wrapper']['element'] = $column['wrapper']['element'] ?? 'a';
+    $column['wrapper']['target'] = $column['wrapper']['target'] ?? '_blank';
+    $column['wrapper']['href'] = asset('/').$column['value'];
```

Comment:
I think the asset here needs to be made the same way we do on the field, check if there is a disk etc.

Reply:
Should I call like 
asset(\Storage::disk($field['disk'])->url(Arr::get($field, 'prefix', '').$column['value'])))

Reply:
I have tried this but no luck. May be we need a quick call to sort this.